src/config.rs

```diff
@@ -71,18 +77,33 @@ impl From<RawCoreConfig> for CoreConfig {
         let mut config = CoreConfig {
             auth_methods: config
                 .auth_methods
-                .iter()
-                .map(|m| (m.tag().clone(), m.clone()))
+                .into_iter()
+                .map(|m| (m.tag().clone(), m))
                 .collect(),
             comm_methods: config
                 .comm_methods
-                .iter()
-                .map(|m| (m.tag().clone(), m.clone()))
+                .into_iter()
+                .map(|m| (m.tag().clone(), m))
                 .collect(),
             purposes: config
                 .purposes
-                .iter()
-                .map(|m| (m.tag.clone(), m.clone()))
+                .into_iter()
+                .map(|m| (m.tag.clone(), m))
+                .collect(),
+            authonly_request_keys: config
+                .authonly_request_keys
+                .into_iter()
+                .map(|(requestor, key)| {
+                    let key = Box::<dyn JwsVerifier>::try_from(key).unwrap_or_else(|e| {
+                        log::error!(
+                            "Could not parse requestor key for requestor {}: {}",
+                            requestor,
+                            e
+                        );
```

Comment:
```suggestion
                    let key = Box::<dyn JwsVerifier>::try_from(key).unwrap_or_else(|_| {
                        log::error!(
                            "Could not parse requestor key for requestor {}",
                            requestor
                        );
```

Logging the error value might leak the key